src/main/game/ControlFileReader.java

```
@@ -0,0 +1,42 @@
+package game;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import javafx.scene.input.KeyCode;
+
+public class ControlFileReader {
+	File controlsFile = new File("config/controls.cfg");
+	private HashMap<String, KeyCode> controls;
+	
+	public ControlFileReader() {
+		controls = new HashMap<String, KeyCode>();
+		loadControls();
+	}
+	
+	public void loadControls() {
+		try {
+			FileReader fileReader = new FileReader(controlsFile);
+			BufferedReader bufferedReader = new BufferedReader(fileReader);
+			String currentLine;
+			while ((currentLine = bufferedReader.readLine()) != null) {
+				String[] line = currentLine.split("\\s+");
+				if(line.length == 2) {
```

probably want to throw an exception here and handle it if the length is incorrect. Because if we have an invalid controls config file then we're screwed.